modules/lnd.nix

```diff
@@ -0,0 +1,136 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+  
+let
+  nix-bitcoin-services = pkgs.callPackage ./nix-bitcoin-services.nix { };
+  cfg = config.services.lnd;
+  bitcoindDataDir = config.services.bitcoind.dataDir;
+  configFile = pkgs.writeText "lnd.conf" ''
```

Comment:
These are reasonable default options, but if  we should have extraoptions to allow people to easily add their own (see my module). I can also add this after this is merged.